src/test/standardTest.node.ts

```diff
@@ -16,15 +16,14 @@ const workspacePath = process.env.CODE_TESTS_WORKSPACE
 const extensionDevelopmentPath = process.env.CODE_EXTENSIONS_PATH
     ? process.env.CODE_EXTENSIONS_PATH
     : EXTENSION_ROOT_DIR_FOR_TESTS;
-const isRunningSmokeTests = process.env.TEST_FILES_SUFFIX === 'smoke.test*';
 const isRunningVSCodeTests = process.env.TEST_FILES_SUFFIX?.includes('vscode.test');
 setTestExecution(true);
 
 function requiresPythonExtensionToBeInstalled() {
     if (process.env.VSC_JUPYTER_CI_TEST_DO_NOT_INSTALL_PYTHON_EXT) {
         return;
     }
-    return isRunningVSCodeTests || isRunningSmokeTests;
+    return isRunningVSCodeTests || IS_SMOKE_TEST || IS_PERF_TEST;
```

Comment:
I believe IS_SMOKE_TEST and IS_PERF_TEST are functions now. I should have changed the case to make that more obvious.